support-frontend/assets/components/marketingConsent/marketingConsentWithCheckbox.jsx

```diff
@@ -92,6 +93,23 @@ function MarketingButton(props: ButtonPropTypes) {
   );
 }
 
+const createRegionalCopy = (titleFreq: string, msgFreq: string): { titleFreq: string, msgFreq: string } => ({
```

Comment:
this works - but just wondering if it might make sense to form the full string at this point since the function implies it returns copy rather than an object? it's completely up to you, though.